quickstart/README.md

```diff
@@ -35,11 +35,16 @@ This document describes how to quickly get started with Hosting Control Planes a
 
 ## What it does
 1. Enables preview_hypershift
-2. Creates a `local-cluster` `managedCluster` for the OpenShift cluster you are installing to
-3. Imports the `local-cluster`
-4. Applies the Hosting Service Cluster addon (Hypershift) to the `local-cluster` (Hub)
+2. Creates a `local-cluster` `managedCluster` for the OpenShift cluster you are installing to if it doesn't
```

Comment:
Something is missing here, either:
1. Creates a `local-cluster` `managedCluster`, if it doesn't already exist
2. Creates a `local-cluster` `managedCluster` to be used as the Hosting Service Cluster, if it doesn't already exist